Lib/test/test_os.py

```diff
@@ -1837,9 +1837,10 @@ def test_win32_mkdir_700(self):
         os.mkdir(path, mode=0o700)
         out = subprocess.check_output(["cacls.exe", path, "/s"], encoding="oem")
         os.rmdir(path)
+        out = out.strip().rsplit(" ")[1]
```

Comment:
What if the patch contains a space? Maybe use a negative index to count from the end?

Reply:
Good point, but you could also just limit the number of splits.

```suggestion
        out = out.strip().rsplit(" ", 1)[1]
```

Reply:
What if a path contains two spaces? :-) I don't think that the number of splits should be limited. Just pick the last item, or -2, I didn't test.

Reply:
It's an `rsplit()` to get the SDDL string from the end, which will not contain spaces. "cacls.exe" prints the path, a space, and the SDDL string. For example:

```text
>>> os.mkdir('C:\\Temp\\spam', 0o700)
>>> os.system('cacls C:\\Temp\\spam /s')
C:\Temp\spam "D:P(A;OICI;FA;;;SY)(A;OICI;FA;;;BA)(A;OICI;FA;;;OW)"
```

The test compares the SDDL string "D:P(A;OICI;FA;;;SY)(A;OICI;FA;;;BA)(A;OICI;FA;;;OW)". This describes a [P]rotected [D]ACL that [A]llows [F]ull-[A]ccess to the directory, and files (OI) and directories (CI) contained within it, to the accounts [SY]stem, [B]uiltin-[A]dministrators, and [OW]ner-Rights.

Reply:
Oh right, I missed the R in rsplit() :-)